BlueprintUI/Sources/Internal/Metatype.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+/// A wrapper to make metatypes easier to work with, providing Equatable, Hashable, and
+/// CustomStringConvertible.
+struct Metatype: Hashable, CustomStringConvertible {
```

Comment:
Love the way this pans out.

Reply:
(Overnight thought: I do wonder if it's worth comparing the old and new hashing and == speeds; since this is in some pretty hot paths to verify nothing much changed)